README.md

```diff
@@ -1,6 +1,6 @@
-# "Microverse Portfolio" project
+# "Hello Microverse (repetition)" project
 
-> This is my mobile version of "Portfolio" project based on the Microverse student's template.
+> Hello Microverse repetition project
 
 ![screenshot](app_screenshot.png)
 
```

Comment:
- [ ] Please, try to fix the display of the screenshot. Actually, it is not displaying your current project, aka "Hello Microverse". Your screenshot comes from the MICROVERSE template. You need just to add your own.